marquez_client/client.py

```diff
@@ -181,7 +185,7 @@ def list_job_runs(self, job_name, limit=None,
         )
 
     def mark_job_run_as_running(self, run_id):
```

Comment:
maybe renaming it to: mark_job_run_as_started
to be more consistent

Reply:
fixed.